simtools/simtel/simtel_runner_ray_tracing.py

```diff
@@ -237,7 +242,11 @@ def _checkRunResult(self, runNumber=None):
 
     def _isPhotonListFileOK(self):
         """Check if the photon list is valid,"""
+        nLines = 0
         with open(self._photonsFile, "r") as ff:
-            nLines = len(ff.readlines())
+            for _ in ff:
```

Comment:
Looks definitely more complicated - any particular reason why the old implementation didn't work?

Reply:
It was significantly slower, it read the entire file (~2 million lines) just to check if it has more than a 100 lines.